src/App.vue

```diff
@@ -42,6 +43,15 @@ export default {
     ...mapState('aeternity', ['sdk']),
   },
   async created() {
+    window.addEventListener('unhandledrejection', (error) => {
```

Comment:
Extract this handler to a new store plugin to don't waste `App.vue`. Also, add a handler for a usual exception as well.

Reply:
great thanks a lot for that catch! use plugin is much better